src/Options.cs

```diff
@@ -1065,17 +1065,19 @@ private static IEnumerable<string> GetLines (string description)
 			}
 			int length = 80 - OptionWidth - 1;
 			int start = 0, end;
+			bool isWhiteSpaceAndNotLastChar;
 			do {
 				end = GetLineEnd (start, length, description);
 				char c = description [end-1];
-				if (char.IsWhiteSpace (c))
+				isWhiteSpaceAndNotLastChar = char.IsWhiteSpace(c) && end > start;

```

Comment:
Suggestion: Define and assign the variable in a single statement:
`bool isWhiteSpaceAndNotLastChar = char.IsWhiteSpace(c) && end > start;`

Then the line `bool isWhiteSpaceAndNotLastChar;` above can be dropped.

Reply:
Sounds good to me.
Thanks for the quick response, btw!